dspace/config/item-submission.xml

```diff
@@ -149,6 +149,13 @@
 	        <type>cclicense</type>
         </step-definition>
 
+		<!-- This optional step may allow the user to select access conditions on the item -->
+        <step-definition id="itemAccessConditions">
```

Comment:
Could we add a commented out example configuration for this new step to the `traditional` submission-process?  That'd make it easier for people to find this new step & it's similar to what we had for 6.x out-of-the-box.  It could be right around here: https://github.com/DSpace/DSpace/blob/main/dspace/config/item-submission.xml#L227  And be something like:
```
<!-- Uncomment this step to allow the user to embargo or access restrict the entire item -->
<!-- <step id="itemAccessConditions"/> -->
```
```